packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -2,111 +2,73 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import * as React from 'react';
+
+import React from 'react';
```

Comment:
can you try to minimize the diff?
![](https://github.com/iTwin/iTwinUI/assets/9084735/5cfb2449-2c90-4a98-bc9b-d04b1dbcc12d)

other than order, the big thing is `import * as React` (which should be preferred over `import React`).

Reply:
Noted, I reorganized the imports